src/main/java/hudson/plugins/sidebar_link/ProjectLinks.java

```diff
@@ -56,10 +56,8 @@ public ProjectLinks(List<LinkAction> links) {
     public List<LinkAction> getLinks() { return links; }
 
     @Override
-    public Collection<? extends Action> getJobActions(AbstractProject<?,?> job) {
```

Comment:
I could not find a case that compatibility with JobDSL is broken (used JobDSL `v1.64`,  Sidebar-Link `v1.9.1` and the snapshot)